x/chainlet/keeper/ccv.go

```diff
@@ -71,6 +69,14 @@ func (k *Keeper) ForcePendingVSC(ctx sdk.Context) {
 		// Send the queued VSC packet immediately
 		ctx.Logger().Info(fmt.Sprintf("force-sending queued VSC packets to a new chainlet %s", chainId))
 		k.providerKeeper.SendVSCPacketsToChain(ctx, chainId, channelId)
```

Comment:
This returns an `error` in latest interchain-accounts repo, we should check for it